plugin/src/utils/run-steps.ts

```diff
@@ -58,10 +71,14 @@ const runSteps = async (steps, helpers, pluginOptions, apiName) => {
   }
 }
 
-const runApiSteps = (steps, apiName) => async (helpers, pluginOptions) =>
-  runSteps(steps, helpers, pluginOptions, apiName)
+const runApiSteps = (steps: Step[], apiName: string) => async (
+  helpers: GatsbyNodeApiHelpers,
+  pluginOptions: IPluginOptions
+): Promise<void> => runSteps(steps, helpers, pluginOptions, apiName)
 
-const runApisInSteps = (nodeApis) =>
+const runApisInSteps = (nodeApis: {
+  [apiName: string]: Step | Step[]
+}): { [apiName: string]: Promise<void> } =>
```

Comment:
This actually doesn't return void :) It returns a Gatsby config object with node api names as keys and a function as a value which runs all the steps that are passed into it.

Reply:
Oh 🤦 lol that's what you typed it as! Ignore this :)